src/main/resources/com/google/api/codegen/go/discovery_fragment.snip

```diff
@@ -147,15 +147,15 @@
     {@requestName()} := {@methodCall(method)}
     if err := {@requestName()}.Pages(ctx, func(page *{@context.getApi.getName}.{@responseType}) error {
       @if context.isMapField(responseTypeRef, resourceField.getName)
-        @let keyName = context.keyTypeName(resourceField), \
-             valueName = context.valueTypeName(resourceField)
+        @let keyName = unclashVar(context.keyTypeName(resourceField)), \
```

Comment:
Just to make sure we remember, add a TODO in `unclash`var to have better suffixes than just digits when porting to MVVM.